hummingbot/connector/exchange/genius_yield/genius_yield_exchange.py

```diff
@@ -167,24 +174,20 @@ async def _place_order(self,
         type_str = GeniusYieldExchange.genius_yield_order_type(order_type)
         side_str = CONSTANTS.SIDE_BUY if trade_type is TradeType.BUY else CONSTANTS.SIDE_SELL
         symbol = await self.exchange_symbol_associated_to_pair(trading_pair=trading_pair)
-        api_params = {"symbol": symbol,
-                      "side": side_str,
-                      "quantity": amount_str,
-                      "type": type_str,
-                      "newClientOrderId": order_id}
-        if order_type is OrderType.LIMIT or order_type is OrderType.LIMIT_MAKER:
-            price_str = f"{price:f}"
-            api_params["price"] = price_str
-        if order_type == OrderType.LIMIT:
-            api_params["timeInForce"] = CONSTANTS.TIME_IN_FORCE_GTC
+        api_params = {
+            "offer_token": symbol.split('_')[0],
+            "offer_amount": str(amount),
+            "price_token": symbol.split('_')[1],
+            "price_amount": str(price)
+        }
 
         try:
             order_result = await self._api_post(
                 path_url=CONSTANTS.ORDER_PATH_URL,
                 data=api_params,
                 is_auth_required=True)
-            o_id = str(order_result["orderId"])
-            transact_time = order_result["transactTime"] * 1e-3
+            o_id = str(order_result["transaction_id"])
+            transact_time = self._time_synchronizer.time()
```

Comment:
Need to check wheter we can use tx id as means of an orders id. Contact api does not have a seperate order id field it seems.